edge-hub/src/Microsoft.Azure.Devices.Routing.Core/endpoints/StoringAsyncEndpointExecutor.cs

```diff
@@ -1,143 +1,144 @@
-// Copyright (c) Microsoft. All rights reserved.
-namespace Microsoft.Azure.Devices.Routing.Core.Endpoints
-{
-    using System;
-    using System.Collections;
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Routing.Core.Endpoints

```

Comment:
Did some formatting change here? Can you try to revert it?

Reply:
I think VS overwrote the CLRF formatting, but a bunch of stuff actually changed in this file, lemme see what I can do to fix it.

Reply:
okay, reverted this file and then manually re-applied the changes, it should now be fixed.